README.md

```diff
@@ -198,6 +198,42 @@ const encryptDecryptFunction = async() => {
 encryptDecryptFunction()
 ```
 
+Encrypt with multiple public keys:
+
+```js
+const pubkeys = [`-----BEGIN PGP PUBLIC KEY BLOCK-----
+...
+-----END PGP PUBLIC KEY BLOCK-----`, 
+`-----BEGIN PGP PUBLIC KEY BLOCK-----
+...
+-----END PGP PUBLIC KEY BLOCK-----`
+const privkey = `-----BEGIN PGP PRIVATE KEY BLOCK-----
+...
+-----END PGP PRIVATE KEY BLOCK-----` //encrypted private key
+const passphrase = `yourPassphrase` //what the privKey is encrypted with
+const message = 'Hello, World!'    // input as Message object
+
+async encryptWithMultiplePublicKeys(pubkeys, privkey, passphrase, message) {
+    const privKeyObj = (await openpgp.key.readArmored(privkey)).keys[0]
+    await privKeyObj.decrypt(passphrase)
+    
+    pubkeys = pubkeys.map(async (key) => {
+    	return (await openpgp.key.readArmored(key)).keys[0]
+    });
+    
+    const options = {
+        message: openpgp.message.fromText(message),       
+        publicKeys: pubkeys,           				  // for encryption
```

Comment:
From line 220, `pubkeys` is array of `Promise`, not array of `Key`.
In this case, line 226 should be `publicKeys: await Promise.all(pubkeys)`

Reply:
Helllo, I will check it and fix if here is the error